example/context/app.js

```diff
@@ -22,13 +26,13 @@ var Color = loopback.createModel('color', { 'name': String });
 Color.beforeRemote('**', function(ctx, unused, next) {
   // Inside LoopBack code, you can read the property from the context
   var ns = loopback.getCurrentContext();
-  console.log('Request to host', ns && ns.get('host'));
+  g.log('Request to host', ns && ns.get('host'));
```

Comment:
`console.log` with multiple strings should be replaced by `g.f` with place holders.